packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -144,6 +147,92 @@ async function handleGrant(releaseGoldConfig: any, currGrant: number) {
   }
 }
 
+async function checkBalance(releaseGoldConfig: any) {
+  const weiAmountReleasedPerPeriod = new BigNumber(
+    web3.utils.toWei(releaseGoldConfig.amountReleasedPerPeriod.toString())
+  )
+  const grantTotal = weiAmountReleasedPerPeriod
```

Comment:
Can we add 1 cGLD here for tx fees?

Maybe call this variable `grantDeploymentCost` instead of `grantTotal` to reflect that